README.md

```diff
@@ -6,27 +6,28 @@ To build the flash loader:
 
 ```bash
 $ cargo build --release --feature $(CHIP_NAME) --target $(RUST_TARGET) # builds the flashing stub
-$ target-gen elf target/$(RUST_TARGET)/release/esp-flashloader output/esp32.yaml --update --name $(CHIP_NAME)-flashloader
+$ target-gen elf target/$(RUST_TARGET)/release/esp-flashloader output/$(CHIP_NAME).yaml --update --name $(CHIP_NAME)-flashloader
 ```
 
 Example for the updating the `esp32c3` flash algorithm.
 
 ```bash
 $ cargo build --release --features esp32c3 --target riscv32imc-unknown-none-elf
-$ target-gen elf target/riscv32imc-unknown-none-elf/release/esp-flashloader output/esp32.yaml --update --name esp32c3-flashloader
+$ target-gen elf target/riscv32imc-unknown-none-elf/release/esp-flashloader output/esp32c3.yaml --update --name esp32c3-flashloader
 ```
 
 ## Chip support
 
-|name   |supported|
-|-------|---------|
-|esp32c3|   Y     |
-|esp32c6|   Y     |
+| name    | supported |
+| ------- | --------- |
+| esp32c2 | Y         |
+| esp32c3 | Y         |
```

Comment:
Oops I forgot to add h2 to this list, could you add it in this PR?

Reply:
Added and just rebased